DRODLib/BuildUtil.h

```diff
@@ -38,7 +38,7 @@ class BuildUtil
 public:
 	static bool bIsValidBuildTile(const UINT wTileNo);
 
-	static void BuildTilesAt(CDbRoom& room, const UINT tile, UINT x, UINT y, const UINT w, const UINT h, const bool bAllowSame, CCueEvents& CueEvents);
+	static bool BuildTilesAt(CDbRoom& room, const UINT tile, UINT x, UINT y, const UINT w, const UINT h, const bool bAllowSame, CCueEvents& CueEvents);
```

Comment:
Pardon, I'm unclear on how returning a bool here is related to this PR.  Can you clarify for me?

Reply:
Glad to! Previously `BuildUtil::BuildTileAt()` and `BuildUtil::BuildTilesAt()` (note plurality) were completely separate, duplicating some trivial logic which at that time was fine. Now, because bridge processing is supposed to happen at the end and I wanted to avoid duplicating logic, I've made the former just use the latter internally. But the single-tile one returned a boolean value, the multi-tile had to be updated to return some kind of information.

The bool return is used by Citizens to know if the build was successful and they can continue or if they need to wait.

Another option would be to modify Multi-tile method to instead return an UINT with the number of tiles that were actually updated, which could potentially be a bit more useful in the future.

Reply:
Actually I'll do the change with UINT and add some docs

Reply:
I appreciate the explanation.  Okay, makes perfect sense.
FYI, the BuildTilesAt return semantic works slighting differently in RPG (for 1.2.8), but it should be fine to keep as you're implementing it here for 5.1.1, as RPG doesn't have any citizen building.